guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -1,11 +1,34 @@
 [id="katello-release-notes"]
 = Katello {KatelloVersion} Release Notes
 
+[id="katello-upgrade-warnings"]
+== Upgrade Warnings
+
+=== Katello-Agent has been removed.
+Katello-Agent has been removed from Katello and is no longer available via web UI, API, or CLI.
+katello-agent will be removed by upgrading, REX alternative will not be installed as part of the upgrade.
+
 [id="katello-headline-features"]
 == Headline Features
 
-[id="katello-upgrade-warnings"]
-== Upgrade Warnings
+=== Pulpcore and plugins upgraded to Pulpcore 3.28.
```

Comment:
We always write a bit of text under a header, explaining why it is a headline feature and why they want it.